File: authentication/authentication/src/main/java/com/example/authentication/model/Order.java
package com.example.authentication.model;

import com.example.authentication.entity.TransactionEntity;
import com.example.authentication.entity.UserEntity;

import java.time.LocalDateTime;

public class Order {
    private Long orderID;
    private UserEntity user;
    private TransactionEntity transaction;
    private String orderDate;
    private String orderStatus;
    private Double totalAmount;
    private LocalDateTime createAt;
    private LocalDateTime updateAt;
}

File: Joomia/app/src/main/java/com/kanyideveloper/joomia/feature_products/domain/model/Order.kt
package com.kanyideveloper.joomia.feature_products.domain.model

import com.kanyideveloper.joomia.feature_cart.domain.model.Transaction
import com.kanyideveloper.joomia.feature_profile.domain.model.User

data class Order (
    val orderID: Int,
    val userID: Int,
    val transactionID: Int,
    val orderDate: String,
    var orderStatus: String,
    val totalAmount: Double
)